viewer/viewer.js

```diff
@@ -5112,11 +5355,23 @@ function buildConnections(req, res, cb) {
   let fdst                 = req.query.dstField;
   let minConn              = req.query.minConn || 1;
 
+  // get the requested fields
+  let fields = ['totBytes', 'totDataBytes', 'totPackets', 'node'];
+  if (req.query.fields) { fields = req.query.fields.split(','); }
+
+  let options;
+  if (req.query.cancelId) { options = { cancelId: `${req.user.userId}::${req.query.cancelId}` }; }
+
```

Comment:
these lines were moved up here from down below (right before they were previously used) since i'm passing them into some of the functions above now. they don't rely on any computed values, just input.